main.go

```diff
@@ -148,6 +147,35 @@ func getInactiveMilestones(baseURL string, token string, projectID string) ([]mi
 	return getMilestones(baseURL, token, projectID, state)
 }
 
+func reactivateClosedMilestones(milestones map[string]milestone, baseURL string, token string, projectID string) error {
+	client := &http.Client{}
+	params := url.Values{}
+	params.Add("state_event", "activate")
+	for _, v := range milestones {
+		milestoneID := v.ID
+		strURL := []string{baseURL, "/projects/", projectID, "/milestones/", milestoneID}
+		URL := strings.Join(strURL, "")
+
+		// Overwrite state information in URL
+		u, _ := url.Parse(URL)
+		q := u.Query()
+		q.Set("state_event", "activate")
+		u.RawQuery = q.Encode()
+		req, err := http.NewRequest("PUT", u.String(), nil)
```

Comment:
Interesting solution. Any specific idea, why this works better than the solution we use for POST? Difference in PUT and POST? You can take some time to research the reasoning behind that.